src/applications/simple-forms/shared/config/submit-transformer.js

```diff
@@ -1,12 +1,11 @@
-import { transformForSubmit as formsSystemTransformForSubmit } from 'platform/forms-system/src/js/helpers';
+import {
+  replaceEscapedCharacters,
+  transformForSubmit as formsSystemTransformForSubmit,
+} from 'platform/forms-system/src/js/helpers';
 
 const escapedCharacterReplacer = (_key, value) => {
   if (typeof value === 'string') {
-    return value
-      .replaceAll('"', "'")
-      .replace(/(?:\r\n|\n\n|\r|\n)/g, '; ')
-      .replace(/(?:\t|\f|\b)/g, '')
-      .replace(/\\(?!(f|n|r|t|[u,U][\d,a-fA-F]{4}))/gm, '/');
+    return replaceEscapedCharacters(value);
   }
```

Comment:
**QUESTION/CONSIDER**
I see this wrapper function repeated a few times still. Is it worth it to try to DRY that up as well? Does it make sense adding `escapedCharacterReplacer` to the platform helper file as well?

Reply:
So, this was me again trying to disturb as little as possible, but now that we have the `replaceEscapedCharacters` option in the shared `transformForSubmit` function that all of these transformers use, we can just call set that option and get rid of the whole wrapper function.